### PR TITLE
Added support for negative mode deconvolution

### PR DESCRIPTION
**Pull request by nbollis**
Summary: Added support for negative mode deconvolution by adding an additional field to the deconvolution parameters. 

- The major change to enable negative mode is the if else structure (lines 77-86) of the ClassicDeconvolutionAlgorithm class

- Code was tested with a known RNA standard. 

- Min and max assumed charge state from the classic deconvolution parameters to the base deconvolution parameters. This is due to all decon methods needing to know a minimum and maximum charge state to perform their calculations. 

- Isotopic envelope needed to be adjusted slightly to take the absolute value of the charge in two cases. 

- Somebody also switched Test project to AnyCPU, so I switched it back to x64.

### CODE REVIEW

File: mzLib/MassSpectrometry/Deconvolution/Deconvoluter.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ namespace MassSpectrometry
     public enum DeconvolutionType
     {
         ClassicDeconvolution,
-        AlexDeconvolution,
+        ExampleNewDeconvolution,
```
Comment (trishorts) [inline mzLib/MassSpectrometry/Deconvolution/Deconvoluter.cs:14]: Lets make this more informative

File: mzLib/MassSpectrometry/MzSpectra/IsotopicEnvelope.cs [hunk 2]
```diff
@@ -35,7 +36,7 @@ public IsotopicEnvelope(List<(double mz, double intensity)> bestListOfPeaks, dou
 
         public double GetMostAbundantObservedIsotopicMass(List<(double mz, double intensity)> peaks, int charge)
         {
-            return peaks.MaxBy(p => p.intensity).mz * charge;
+            return peaks.MaxBy(p => p.intensity).mz * Math.Abs(charge);
```
Comment (trishorts) [inline mzLib/MassSpectrometry/MzSpectra/IsotopicEnvelope.cs:39]: nice

**Review by trishorts — APPROVED** [review]
looks good see comments.

File: mzLib/MassSpectrometry/Deconvolution/Deconvoluter.cs (hunk 1 above)
Comment (Alexander-Sol) [reply mzLib/MassSpectrometry/Deconvolution/Deconvoluter.cs:14]: I'd like my name to still be included tho

**Review by Alexander-Sol — DISMISSED** [review]
lgtm

**Review by trishorts — APPROVED** [review]

**Review by elaboy — APPROVED** [review]

**Review by Alexander-Sol — APPROVED** [review]

**Pull request merged**